### ISSUE TITLE
fail positive in no-invalid-position-declaration

### ISSUE THREAD

**Issue by Mister-Hope** [stylelint-scss/stylelint-scss#1189]
<img width="1117" height="261" alt="Image" src="https://github.com/user-attachments/assets/a6590b80-8e4a-4df1-82c5-930bc88ad621" />

```scss
@use 'variables' as *;

@mixin content-wrapper {
  max-width: var(--content-width);
  margin: 0 auto;
  padding: 2rem 2.5rem;

  @media (max-width: $MQNarrow) {
    padding: 2rem;
  }
}
```

```
Unexpected invalid position declaration (no-invalid-position-declaration)Stylelint[no-invalid-position-declaration](https://stylelint.io/user-guide/rules/no-invalid-position-declaration)
```

This is a real error in css files so stylelint is right, which it has been reported as problem in scss files, and stylelint-scss shall silence it.

**Comment by Mister-Hope** [comment]
Hi, some update ping. I am not pushing anyone, but could there at least be some issue tagging or review comments being made with this issue? Half a year has passed, and we are not receiving some feedbacks.
